<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>geo_companion_dock_widget</class>
    <widget class="QDockWidget" name="geo_companion_dock_widget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>341</width>
                <height>335</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>1</verstretch>
            </sizepolicy>
        </property>
        <property name="windowTitle">
            <string>MapsIndoors Editor</string>
        </property>
        <widget class="QWidget" name="dockWidgetContents">
            <property name="enabled">
                <bool>true</bool>
            </property>
            <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>1</horstretch>
                    <verstretch>1</verstretch>
                </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
                <property name="sizeConstraint">
                    <enum>QLayout::SetMinimumSize</enum>
                </property>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <layout class="QVBoxLayout" name="verticalLayout_3">
                                <property name="leftMargin">
                                    <number>10</number>
                                </property>
                                <property name="rightMargin">
                                    <number>10</number>
                                </property>
                                <item>
                                    <widget class="QLabel" name="icon_label">
                                        <property name="text">
                                            <string/>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLabel" name="version_label">
                                        <property name="text">
                                            <string>0.0.0</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QVBoxLayout" name="verticalLayout_2">
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                                        <item>
                                            <widget class="QLabel" name="label">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="text">
                                                    <string>Solution:</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QComboBox" name="solution_combo_box">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="editable">
                                                    <bool>true</bool>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="solution_reload_button">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="text">
                                                    <string>Load Solutions</string>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                                        <item>
                                            <widget class="QLabel" name="label_2">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="text">
                                                    <string>Venue:</string>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QComboBox" name="venue_combo_box">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="editable">
                                                    <bool>true</bool>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="venue_reload_button">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="text">
                                                    <string>Load Venues</string>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                                <item>
                                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                                        <item>
                                            <spacer name="horizontalSpacer_3">
                                                <property name="orientation">
                                                    <enum>Qt::Horizontal</enum>
                                                </property>
                                                <property name="sizeHint" stdset="0">
                                                    <size>
                                                        <width>40</width>
                                                        <height>20</height>
                                                    </size>
                                                </property>
                                            </spacer>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="sync_button">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="text">
                                                    <string>Fetch Venue</string>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QPushButton" name="revert_button">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>Revert Venues</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="changes_label">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>Status</string>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="upload_button">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="text">
                                    <string>Sync Venues</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QScrollArea" name="scrollArea">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <property name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Sunken</enum>
                        </property>
                        <property name="sizeAdjustPolicy">
                            <enum>QAbstractScrollArea::AdjustIgnored</enum>
                        </property>
                        <property name="widgetResizable">
                            <bool>true</bool>
                        </property>
                        <property name="alignment">
                            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                        </property>
                        <widget class="QWidget" name="layoutWidget">
                            <property name="geometry">
                                <rect>
                                    <x>0</x>
                                    <y>0</y>
                                    <width>16</width>
                                    <height>16</height>
                                </rect>
                            </property>
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>0</width>
                                    <height>0</height>
                                </size>
                            </property>
                            <layout class="QGridLayout" name="entry_point_grid" rowstretch="0" columnstretch="0">
                                <property name="sizeConstraint">
                                    <enum>QLayout::SetNoConstraint</enum>
                                </property>
                                <property name="leftMargin">
                                    <number>6</number>
                                </property>
                                <property name="topMargin">
                                    <number>6</number>
                                </property>
                                <property name="rightMargin">
                                    <number>6</number>
                                </property>
                                <property name="bottomMargin">
                                    <number>6</number>
                                </property>
                                <property name="spacing">
                                    <number>6</number>
                                </property>
                            </layout>
                        </widget>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
